app/controllers/orga/conferences_controller.rb

```diff
@@ -1,9 +1,14 @@
 class Orga::ConferencesController < Orga::BaseController
 
+  def new
+    @conference = Conference.new
```

Comment:
The forms used for `new` and `edit` currently use the `conference` helper. Since it seems like there's a `@conference` previously initialized by the controller method anyways, I'd vote for getting rid of the `helper_method` macro and instead use the normal `@conference` variable in the views to spare this weird back and forth between view and controller methods.